typing/typecore.mli

```diff
@@ -282,6 +284,10 @@ val type_object:
 val type_package:
   (Env.t -> Parsetree.module_expr -> package ->
    Typedtree.module_expr * package) ref
+(* Forward declaration, to be filled in by Typemod.check_package_closed *)
+val check_package_closed:
```

Comment:
I think it would be nice to have a documentation comment on this declaration, since the `Typemod` implementation is not exposed in interface file (and I have to check which version of closed this function was checking).